tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -28,10 +28,19 @@
 #
 # A map: function name => name of the argument that all outputs are view of
 
-VIEW_FUNCTIONS_WITH_METADATA_CHANGE = ['view_as_complex', 'view_as_real', '_conj', '_neg_view']
+VIEW_FUNCTIONS_WITH_METADATA_CHANGE = [
+    'view_as_complex',
+    'view_as_real',
+    '_conj',
+    '_neg_view',
+    'matrix_H',
+    'mH',
```

Comment:
you probably don't need these since they are composite ops and we wouldn't wanna replay the op if these functions are called on real tensors

Reply:
Should I leave `adjoint`?

Reply:
I assume from the discussion below that I should also delete `adjoint`.